### DISCUSSION TITLE
Collapse Groups on Grid

### DISCUSSION THREAD

**Discussion by Ampurcell** [cortexrd/Knack-Toolkit-Library#388] [Ideas]
Hello! 

Hope everyone is doing fabulously! 
Random idea - Would it be possible to get a collapse arrow next to "group" headers in a grid view? 

![Screenshot 2024-10-07 at 12 41 28 AM](https://github.com/user-attachments/assets/2d424e5a-c0a8-4bd6-9321-50b1b961d49d)

**Comment by cortexrd** [comment]
That would be cool and useful!  But not so trivial to implement...
I'll think about it but can't commit to anything right now.

If some coder out there wants to give it a try, please go ahead and submit your work!

Norm
